.github/workflows/cmake.yml

```
@@ -84,3 +84,30 @@ jobs:
 
     - name: Install
       run: cmake --install build --prefix scalapack_install
+
+  freebsd:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Repository checkout
+        uses: actions/checkout@v4
+      - uses: cross-platform-actions/action@9a4355714f90c057476048788b82147e99b7c0f0 # v0.28.0 + fix for 14.3
```

```suggestion
      - uses: cross-platform-actions/action@v0.29
```
Could use this now right?